ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -552,6 +552,22 @@ extension SignalType {
 	}
 }
 
+extension SignalProducerType {
+	/// Merges the given producers into a single `SignalProducer` that will emit all values
+	/// from each of them, and complete when all of them have completed.
+	@warn_unused_result(message="Did you forget to call `start` on the producer?")
+	public static func merge<S : SequenceType where S.Generator.Element == SignalProducer<Value, Error>>(producers: S) -> SignalProducer<Value, Error> {
+		return SignalProducer(values: producers).flatten(.Merge)
+	}
+	
+	/// Merges the given producers into a single `SignalProducer` that will emit all values
+	/// from each of them, and complete when all of them have completed.
+	@warn_unused_result(message="Did you forget to call `start` on the producer?")
+	public static func merge<Value, Error: ErrorType>(producers: SignalProducer<Value, Error>...) -> SignalProducer<Value, Error> {
```

Comment:
Can you add a `...` variant like this to `Signal` as well if we're going to try to even out the interfaces?